docs/user/versions.rst

```diff
@@ -149,12 +149,25 @@ Version Control Support Matrix
 Version warning
 ---------------
 
-This is a banner that appears on the top of every page of your docs that aren't stable or latest.
-This banner has a text with a link redirecting the users to the latest version of your docs.
+A banner can be automatically displayed to notify viewers that there may be
+a more stable version of the documentation available. Specifically:
+
+- When the ``latest`` version is being shown, and there's also a ``stable`` version available,
+  then the banner will remind the viewer that some of the documented features may not yet be
+  available, and suggest that the viewer switch to the ``stable`` version.
+- When a version is being shown that is not the ``stable`` version, and there's a ``stable``
+  version available, then the banner will suggest that the viewer switch to the ``stable`` version
+  to see the newest documentation.
+
```

Comment:
It would be good to add a small image showing the banners in action on each case to easily visualize the feature.

Reply:
Thanks for the review. I ended up using my own custom template for the version banners on my doc project (unrelated reasons), so I don't have a project available to take the screenshots. It may take me some time to prepare a new project with the versioning setup for taking these screenshots.

Reply:
Ok. Don't worry. I can add these screenshot later in another PR. Let's try to finish this one for now. Thanks!